packages/core/src/extension/extension-request-sender.ts

```diff
@@ -0,0 +1,28 @@
+import { RequestSender, Response } from '@bigcommerce/request-sender';
+
+import {
+    ContentType,
+    INTERNAL_USE_ONLY,
+    RequestOptions,
+    SDK_VERSION_HEADERS,
+} from '../common/http-request';
+
+import { Extension } from './extension';
+
+export const EXTENSIONS_API_URL = '/api/storefront/checkout-extensions';
+
+export class ExtensionRequestSender {
+    constructor(private _requestSender: RequestSender) {}
+
+    loadExtensions({ timeout, params }: RequestOptions = {}): Promise<Response<Extension[]>> {
+        return this._requestSender.get(EXTENSIONS_API_URL, {
+            timeout,
+            headers: {
+                Accept: ContentType.JsonV1,
+                'X-API-INTERNAL': INTERNAL_USE_ONLY,
```

Comment:
I remember there was discussion offline, do we need to add this?

Reply:
Removed.